tests/utilities/test_cli.py

```diff
@@ -687,3 +690,63 @@ def __init__(self, optim1: dict, optim2: dict, scheduler: dict):
     assert isinstance(cli.model.optim1, torch.optim.Adam)
     assert isinstance(cli.model.optim2, torch.optim.SGD)
     assert isinstance(cli.model.scheduler, torch.optim.lr_scheduler.ExponentialLR)
+
+
+def test_registries(tmpdir):
+
+    assert CALLBACK_REGISTRIES.available_objects() == [
+        "BackboneFinetuning",
+        "BaseFinetuning",
+        "BasePredictionWriter",
+        "Callback",
+        "EarlyStopping",
+        "GPUStatsMonitor",
+        "GradientAccumulationScheduler",
+        "LambdaCallback",
+        "LearningRateMonitor",
+        "ModelCheckpoint",
+        "ModelPruning",
+        "ProgressBar",
+        "ProgressBarBase",
+        "QuantizationAwareTraining",
+        "StochasticWeightAveraging",
+        "Timer",
+        "XLAStatsMonitor",
+    ]
+
+    class TestModel(BoringModel):
+        def __init__(self):
+            super().__init__()
+
+    callbacks = [
+        dict(
+            class_path="pytorch_lightning.callbacks.Callback",
+            init_args=dict(),
+        ),
+        dict(
+            class_path="pytorch_lightning.callbacks.Callback",
+            init_args=dict(),
+        ),
+    ]
```

Comment:
Can you also test passing a config (dict) where `class_path` and `init_args` are not used?

Reply:
Not sure to fully understand what you mean. Just test with an `init_args` removed right ? If yes, done.